### PR TITLE
Mane Antonyan

### PR DESCRIPTION
**Pull request by mKolibri**
Add html file(registration).
web page in html

### CODE REVIEW

File: Mane_Antonyan/register.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Registration Form</title>
+		</style>
```
Comment (mkhitar) [inline Mane_Antonyan/register.html]: what is this?

File: Mane_Antonyan/register.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+	<head>
```
Comment (mkhitar) [inline Mane_Antonyan/register.html]: please use 4 spaces for indentation.

File: Mane_Antonyan/register.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+		<title>Registration Form</title>
+		</style>
+	</head>
+	<body>
+		<h2>REGISTRATION</h2>
+		<form action="">
+			<fieldset style="width:270px; background-color:#f5f1f1">
+				<legend color="#19a1a4">Sign up</legend> 
+				<form action="">
+					<label style="margin: 20px 0 5px 0; display: block">
```
Comment (mkhitar) [inline Mane_Antonyan/register.html]: Please read about <label> tag:
1. Should wrap a form element (not a span)
2. When you click on label the corrosponding form element is focused (if it's used correctly)

File: Mane_Antonyan/register.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+						<input type=tel placeholder="###" maxlength=3 style="margin-left: 5px; width:30px"></input>
+						<input type=tel placeholder="###" maxlength=3 style="margin-left: 5px; width:30px"></input>
+						<input type=tel placeholder="###" maxlength=3 style="margin-left: 5px; width:30px"></input>
+						<input type=tel placeholder="###" maxlength=3 style="margin-left: 5px; width:30px"></input>
+					</div>
+
+					<div style="margin: 20px 0">
+						<form action="">
+							<input type="radio" name="male" value="male"> Male</input> <br>
+							<input type="radio" name="female" value="female" > Female</input> <br>
```
Comment (mkhitar) [inline Mane_Antonyan/register.html]: Redundant space

File: Mane_Antonyan/register.html [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+							<input type="radio" name="other" value="other" >Other</input>
+							<select name="select" disabled>
+								<option value="volvo">1</option> 
+								<option value="saab">2</option>
+								<option value="mercedes">3</option>
+							</select>
+							</input>
+
```
Comment (mkhitar) [inline Mane_Antonyan/register.html]: Redundant blank line

File: Mane_Antonyan/register.html [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+					<label style="margin: 20px 0 5px 0; display: block">
+						Repeat Password
+						<span style="color: red">*</span>
+					</label>
+					<input name="re-password" pattern="^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!/+-]).{6,40}$" required placeholder="Repeat password" type="password">
+					<div>
+						<input type="submit" value="Send" style="display: inline-block; width: 117px; margin: 10px 6px 0 0;"/>
+						<input type="submit" value="Close" style="display: inline-block; width: 117px; margin: 10px 0 0 0;"/>
+						<script>
+						function myFunction() {
```
Comment (mkhitar) [inline Mane_Antonyan/register.html]: unused function declaration.

File: Mane_Antonyan/register.html [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+					<input name="password" pattern="^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!/+-]).{6,40}$" required placeholder="Enter password" type="password">
+
+					<label style="margin: 20px 0 5px 0; display: block">
+						Repeat Password
+						<span style="color: red">*</span>
+					</label>
+					<input name="re-password" pattern="^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!/+-]).{6,40}$" required placeholder="Repeat password" type="password">
+					<div>
+						<input type="submit" value="Send" style="display: inline-block; width: 117px; margin: 10px 6px 0 0;"/>
+						<input type="submit" value="Close" style="display: inline-block; width: 117px; margin: 10px 0 0 0;"/>
```
Comment (mkhitar) [inline Mane_Antonyan/register.html]: There is no point to have 2 submit buttons doing the same thing with different names.

File: Mane_Antonyan/register.html [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+
+						</form>
+					</div>
+
+
+					<label style="margin: 20px 0 5px 0; display: block">
+						Password
+						<span style="color: red">*</span>
+					</label>
+					<input name="password" pattern="^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!/+-]).{6,40}$" required placeholder="Enter password" type="password">
```
Comment (mkhitar) [inline Mane_Antonyan/register.html]: If there is a format for password it should be clarified for user when they input incorrect formatted password, there should be human readable hint.

**Comment by mKolibri** [comment]
Hi Mkhitar,
thanks for your comments,

I enhanced my code based on your comments and provided new PR.
PR link: https://github.com/Galateyta/ITC-10/pull/86

Regards,
Mane Antonyan

**Pull request closed without merging**